ntc_templates/templates/cisco_nxos_show_hardware_internal_bigsur_all-ports_detail.textfsm

```diff
@@ -0,0 +1,457 @@
+Value Required PORT_NAME (\S+)
+Value Required IF_INDEX (\S+)
+Value INSTANCE_INDEX (\S+)
+Value INSTANCE_SLOT (\S+)
+Value INSTANCE_ASIC (\S+)
+Value ASIC_EPORT (\S+)
+Value LOGICAL_PORT (\S+)
+Value FRONT_PORT (\S+)
+Value STATE ([\S\s]+?)
+Value BIGSUR_EPORT (\S+)
+Value PORT_TYPE (\S+)
+Value PORT_TYPE_DEC (\S+)
+Value TYPE (\S+)
```

Comment:
What is this for? Can we get a bit more?

Reply:
I have no idea what the field bigsur eport is, but he supervisors have a code of 0 and the 40gbe have a code of 1
port_type indicates the string of the hardware type ie 40gbe
port_type_dec is the decimal code for the that type ie 12
I also don't know much about what the type field indicates other than that the sup and 40gbe differ.
I was just doing my best to make sure all of the variables provided in this output were accounted for so anyone else that wanted to use them could.